journal.py

```diff
@@ -135,6 +152,49 @@ def read_entries(request):
     cursor.execute(DB_ENTRIES_LIST)
     keys = ('id', 'title', 'text', 'created')
     entries = [dict(zip(keys, row)) for row in cursor.fetchall()]
+    for entry in entries:
+        entry['text'] = markdown.markdown(entry['text'], extensions=['codehilite', 'fenced_code'])
```

Comment:
Why are you rendering the text here, and then using the markdown rendering filter in the template that renders the response for this view?